src/contracts/storage/tick.rs

```diff
@@ -22,6 +22,21 @@ pub struct Tick {
     pub seconds_outside: u64,
 }
 
+// 131072 / (32 + 128 + 128 + 64) > 372
+pub const POSITION_TICK_LIMIT: usize = 372;
```

Comment:
Calculate it in code. You can easily do it using the `mem::size_of` function. This function is probably from `std`, but this shouldn't be problem